### PR TITLE
test-hammer-cli-foreman-discovery - convert to JJB

### PR DESCRIPTION
**Pull request by komidore64**

### CODE REVIEW

File: puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_hammer_cli_foreman_discovery.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+          type: user-defined
+          name: ruby
+          values:
+            - 2.3
+            - 2.4
+            - 2.5
+    builders:
+      - shell: !include-raw: scripts/test/test_hammer_cli_foreman_discovery.sh
+    publishers:
+      - gemset_cleanup
```
Comment (ehelms) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_hammer_cli_foreman_discovery.yaml]: Should this also have an IRC notification publisher?

Comment (komidore64) [reply puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_hammer_cli_foreman_discovery.yaml]: oops. missed that! yes it should.

Comment (komidore64) [reply puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_hammer_cli_foreman_discovery.yaml]: fixed

**Comment by komidore64** [comment]
added `test_hammer_cli_foreman_discovery_pull_request` because it also references `scripts/test/test_hammer_cli_foreman_discovery.sh`.

**Pull request merged**